airframe-http-recorder/README.md

```diff
@@ -66,13 +66,15 @@ withResource(HttpRecorder.createReplayOnlyServer(recorderConfig)) { server =>
 
 ### Programmable HttpRecorderServer
 
+You can record responses by code instead of recording real interactions with the server.
+
 ```scala
 import wvlet.airframe.http.recorder._
 import wvlet.airframe.control.Control._
 import com.twitter.finagle.http.{Request,Response}
 
 val recorderConfig = HttpRecorderConfig(sessionName="my-recording")
-val response = withResource(HttpRecorder.createProgrammableServer(recorderConfig)) { server =>
+val response = withResource(HttpRecorder.createReplayOnlyServer(recorderConfig)) { server =>
```

Comment:
Using ReplayOnlyServer as ProgrammableServer might be a little weird? InMemoryServer doesn't have variation of creation methods like recordOnly and replayOnly.

Reply:
Ok, so how about renaming APIs as createRecordServer and createReplayServer?

Reply:
Sorry for late reply. That sounds good! I was also thinking that the following namings for consistency:

- createRecordServer
- createServer
- createInMemoryServer

What do you think?

Reply:
I'm ok with these changes!